securesystemslib/keys.py

```diff
@@ -1771,29 +1455,13 @@ def create_rsa_encrypted_pem(private_key, passphrase):
   # Does 'passphrase' have the correct format?
   securesystemslib.formats.PASSWORD_SCHEMA.check_match(passphrase)
 
-  # Raise 'securesystemslib.exceptions.UnsupportedLibraryError' if the following
-  # libraries, specified in 'settings', are unsupported or unavailable:
-  # 'settings.GENERAL_CRYPTO_LIBRARY' and 'settings.RSA_CRYPTO_LIBRARY'.
-  check_crypto_libraries(['rsa', 'general'])
-
   encrypted_pem = None
 
-  # Generate the public and private RSA keys. The PyCrypto module performs
-  # the actual key generation. Raise 'ValueError' if 'bits' is less than 1024
-  # or not a multiple of 256, although a 2048-bit minimum is enforced by
+  # Generate the public and private RSA keys. Raise 'ValueError' if 'bits' is
+  # less than 1024, although a 2048-bit minimum is enforced by
```

Comment:
This does not happen here or in the called `pyca_crypto_keys.create_rsa_encrypted_pem`.